babel.config.js

```diff
@@ -15,12 +15,17 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-module.exports = {
-  presets: [
+module.exports = (api) => {
+  api.cache(true)
+  const presets = [
     '@vue/app'
-  ],
-  plugins: [
+  ]
+  const plugins = [
     ['@babel/plugin-proposal-class-properties', { loose: true }],
+    ['@babel/plugin-proposal-private-methods', { loose: true }],
+    ['@babel/plugin-proposal-nullish-coalescing-operator'],
+    ['@babel/plugin-proposal-optional-chaining'],
     ['babel-plugin-istanbul', { extension: ['.js', '.vue'] }]
   ]
+  return { presets, plugins }
 }
```

Comment:
Different syntax to make it clear that we have an array, and also to access the `api` object and enable the [cache](https://babeljs.io/docs/en/config-files#apicache). It's also possible to access the `NODE_ENV` that way if necessary, and write JS code to enable/disable plug-ins per environment (e.g. I'll probably completely disable everything related to istanbul & coverage when `env === 'production'`, but for another PR)

Reply:
The important new lines here are

```
    ['@babel/plugin-proposal-nullish-coalescing-operator'],
    ['@babel/plugin-proposal-optional-chaining']
```
For the `object?.property` operator.